lib/ttnt/tasks.rb

```diff
@@ -0,0 +1,54 @@
+require 'rake/testtask'
+require 'rake/file_list'
+require 'ttnt/testtask'
+require 'ttnt/test_selector'
+require 'rugged'
+
+# Reference: https://github.com/bundler/bundler/blob/master/lib/bundler/gem_helper.rb
+module TTNT
+  class TaskHelper
+    include Rake::DSL if defined? Rake::DSL
```

Comment:
Thank you! Fixed here: https://github.com/Genki-S/ttnt/commit/53c6f750cfbf69e8cadeeddbdccccadfdd5bbf1a

My understanding is that rake is bundled with ruby so `require 'rake/testtask'` does never fail to define `Rake::DSL`, is this correct?